package.json

```diff
@@ -26,7 +26,7 @@
   "license": "SEE LICENSE IN https://tc39.es/ecma262/#sec-copyright-and-software-license",
   "homepage": "https://tc39.es/ecma262/",
   "dependencies": {
-    "ecmarkup": "^3.19.0"
+    "ecmarkup": "3.20.0"
```

Comment:
```suggestion
    "ecmarkup": "^3.20.0"
```

Reply:
I'd personally prefer pinning a precise version, so that the build output only changes when a human explicitly makes the decision to change it.

(I feel like we've had this conversation already, but I can't find it. Maybe it was with someone else.)

@michaelficarra @syg thoughts?

Reply:
That's what the lockfile already achieves, for those interested in using it, like ecma262. The `^` indicates that ecmarkup follows semver. In other words, pinning it here adds no determinism, because we already have a lockfile.

Reply:
Ah, yeah, lockfile is good enough. Will fix.